packages/web-components/fast-components/src/radio/radio.md

```diff
@@ -0,0 +1,109 @@
+# Radio
+
+## Overview
+An implementation of a radio button as a form-connected web-component. Facilitating single select from a group of visible choices.
+
+### Use Cases
+Used anywhere an author might otherwise use an `input[type="radio"]`. Used to facilitate choice where only one choice is acceptable.
+  
+### Features
+- form association
+- focus delegation
+
+### Risks and Challenges
+We want general feature-parity between this component and an `input[type="radio"]`.
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/radio)
+- [Material UI](https://material-ui.com/components/radio-buttons/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/radio-group/)
+- [Carbon Design](https://www.carbondesignsystem.com/components/radio-button/code)
+- [Ant Design](https://ant.design/components/radio/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/radio)
+- [Office Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/choicegroup)
+- [Windows (UWP)](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/radio-button)
+---
+
+### API
+Extends [form associated custom element](../form-associated/form-associated-custom-element.md).
+
+*Component Name*
+`fast-radio`
+
+*IDL attributes*
+- `checked: boolean`
+  - The current checked state of the radio
+
+*Content attributes*
+- `readonly`
+  - The radio should be submitted with the form but should not be editable.
+- `disabled`
+  - The radio should be disabled from user interaction and will not be submitted with the form data.
+- `value` - Not visible to the user, it's used for form data and to distinguish between other radio buttons of the same name attribute value.
+- `name` - All radio buttons with the same name will operate as a group allowing only 
+one radio within the group to be selected at a time.
+- `checked`
+  - The initial checked value. 
+  
+*Events*
+- `change: CustomEvent`
+  - no custom data
+  - bubbles
+
+### Anatomy and Appearance
+
+```HTML
+<!-- shadow root -->
+<template 
+  role="radio"
+>
+  <label part="label"><slot></slot></label>
+  <div part="radio">
+    <div part="checked-indicator"></div>
```

Comment:
Do we want a slot for the indicator?

Reply:
hmm seems like we might, I'm my defense I did ponder this for a minute. I think we'd need this already for EZ settings. So I will add it.

Reply:
I think we will, similar to checkbox I believe. The checked indicator here can likely be an SVG as well "defaulted" into the slot.

Reply:
done and done